src/main/java/jenkins/plugins/office365connector/DecisionMaker.java

```diff
@@ -51,7 +51,7 @@ public DecisionMaker(Run run, TaskListener listener) {
      * @return <code>true</code> if at least one macro has matched, <code>false</code> otherwise
      */
     public boolean isAtLeastOneRuleMatched(Webhook webhook) {
-        if (webhook.getMacros().isEmpty()) {
+        if (webhook.getMacros() == null || webhook.getMacros().isEmpty()) {
```

Comment:
This problem may happen for unreleased version but for the final it won't happened as macros won't be `null`. Check https://github.com/damianszczepanik/office-365-connector-plugin/commit/e5c613a70bdd8fb717ebcbc6c737e072b74b64b1#diff-4d256ee5f164b70f36b78f29a615087cR157

Then this fix is not needed

Reply:
I just double checked to make sure. Running master / commit a78ca0c16327eed2b0ffdaaf590e27bdac589ca9 , the issue is present.

Reply:
in any case I do not want to add the null check 👎